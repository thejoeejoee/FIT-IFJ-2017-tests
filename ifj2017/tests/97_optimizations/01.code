' constant expression evaluating

scope
	dim counter as integer = 50

	do while counter > 0
		print (10.2 / 98 + (12 \ 3 * 6.85) - 14) / (0 - 5) + (0 + 9 - 8) * ((12+17.21)\(10.8 - 7));
		counter = counter - 1
		print !"\n";
	loop

end scope