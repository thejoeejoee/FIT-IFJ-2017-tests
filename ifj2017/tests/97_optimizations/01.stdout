 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
 9.77918
